### ISSUE TITLE
API Docs

### ISSUE THREAD

**Issue by carlesm** [peterbuga/HASS-sonoff-ewelink#82]
Was checking stuff before submitting a new device (curtain/blinds switches), with additional info and discovered this repo, by the makers of ewelink:

https://github.com/CoolKit-Technologies/apiDocs

Seems they are documenting, at least that seems, the API (although in chinese).

**Comment by peterbuga** [comment]
**WOW! many thanks!**
Thanks to @2016for I've got ahold of a similar API docs version before, but this one seems fresh-er and looks like someone actually is pushing some the pages translated in english too!

**Comment by carlesm** [comment]
Yep, there are pushes of almost all API in english. Forked it, to prevent dissapearances :-D
